src/components/AddPlaidBankAccount.js

```diff
@@ -119,7 +119,7 @@ class AddPlaidBankAccount extends React.Component {
         if (!plaidBankAccounts.length) {
             return (
                 <FullPageOfflineBlockingView>
-                    {(!token || this.props.plaidData.isLoading)
+                    {(this.props.plaidData.isLoading || (!token && !plaidDataErrorMessage))
```

Comment:
I actually completely removed !token [in my PR](https://github.com/Expensify/App/pull/13236/files#diff-2a35d342f7be1cc5c1a8f02937f635a5c7083ec66b64b400027d65087d400e09L122). So that should fix it?

Reply:
Ah, yeah it probably does :/ haha

Reply:
I updated it in case we want to just merge this now while the other PR continues to be reviewed and tested